### DISCUSSION TITLE
Lab A 02-mixing-c-rtl-kernels: No device trace in summary

### DISCUSSION THREAD

**Discussion by piggy0622** [bol-edu/HLS-SOC-Discussions#30] [Q&A] [answered]
I  followed the steps in tutorial for run2 as below:

1. Create the project to generate rtl_kernel_wizard_0.xo successfully.
2. make all TARGET=hw_emu LAB=run2 
3. export XCL_EMULATION_MODE=hw_emu
4. ./host krnl_vadd.hw_emu.xilinx_u50_gen3x16_xdma_5_202210_1.xclbin
5. vitis_analyer xrt.run_summary

What I have already tried is to modify the xrt.ini as shown below. 

<img width="693" alt="image" src="https://user-images.githubusercontent.com/60806185/195495094-844a959c-7ad5-4e41-bd9f-7f888865f226.png">
<img width="406" alt="image" src="https://user-images.githubusercontent.com/60806185/195495243-418a5160-8931-4bea-a7c0-f31a5a674cca.png">

**Comment by alankuo04** [comment, marked as answer]
You can create a new Application project in Vitis and import the kernel and host program files, then run hardware emulation in Vitis, I got the device trace in summary from the method like this.
